src/styles/main.scss

```diff
@@ -1,3 +1,19 @@
-body {
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
 }
+
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+html {
+  font-family: "Roboto", sans-serif;
+}
```

Comment:
styles of html should be in separate file

Reply:
I agree. I'll make changes.